src/napari_micromanager/_gui_objects/_toolbar.py

```diff
@@ -394,3 +191,151 @@ def _add_dock_widget(
             dock_wdg._close_btn = False
         dock_wdg.setFloating(floating)
         return dock_wdg
+
+
+# -------------- Toolbars --------------------
+
+
+class MMToolBar(QToolBar):
+    def __init__(
+        self, title: str, parent: QWidget = None, gb: QGroupBox | None = None
+    ) -> None:
+        super().__init__(title, parent)
+        self.setMinimumHeight(TOOLBAR_SIZE)
+        self.setSizePolicy(QSizePolicy.Policy.Expanding, QSizePolicy.Policy.Fixed)
+        self.setObjectName(f"MM-{title}")
+
+        if isinstance(gb, QGroupBox):
+            self.groupbox = gb
+            gb.setTitle("")
+        else:
+            self.groupbox = QGroupBox()
+            gb_layout = QHBoxLayout(self.groupbox)
+            gb_layout.setContentsMargins(0, 0, 0, 0)
+            gb_layout.setSpacing(2)
+
+        self.groupbox.setStyleSheet(GROUPBOX_STYLE)
+        self.addWidget(self.groupbox)
+
+    def addSubWidget(self, wdg: QWidget) -> None:
+        self.groupbox.layout().addWidget(wdg)
+
+
+class ConfigToolBar(MMToolBar):
+    def __init__(self, parent: QWidget) -> None:
+        super().__init__("Configuration", parent, ConfigurationWidget())
```

Comment:
see `ConfigurationWidget()  `comment above
```suggestion
        super().__init__("Configuration", parent=parent)
        self._wdg = ConfigurationWidget()
        self.addSubWidget(self._wdg)
```
        self.addSubWidget(self._wdg)